.github/workflows/release.yml

```
@@ -300,7 +300,7 @@ jobs:
         run: gon -log-level=debug -log-json "${{ env.GON_CONFIG_PATH }}"
 
       - name: Zip output app bundle
-        run: ditto -c -k ArduinoCreateAgent.app/ ArduinoCreateAgent.app_${{ matrix.arch }}_notarized.zip
+        run: ditto -c -k ArduinoCreateAgent.app/ ArduinoCloudAgent.app_${{ matrix.arch }}_notarized.zip
```

```suggestion
        run: ditto -c -k ArduinoCloudAgent.app/ ArduinoCloudAgent.app_${{ matrix.arch }}_notarized.zip
```
Shouldn't update this too? Along with L289